packages/auth/src/common/AuthErrorStrings.ts

```diff
@@ -9,4 +9,6 @@ export enum AuthErrorStrings {
 	INVALID_MFA = 'Invalid MFA type',
 	EMPTY_CHALLENGE = 'Challenge response cannot be empty',
 	NO_USER_SESSION = 'Failed to get the session because the user is empty',
+	NETWORK_ERROR = 'Network Error',
+	DEVICE_CONFIG = 'Device tracking has not been properly configured in Cognito User Pools',
```

Comment:
Maybe this is more clear 
```suggestion
	DEVICE_CONFIG = 'Device tracking has not been configured in this User Pool'
```